CONTRIBUTING.md

```diff
@@ -47,47 +47,48 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+the extension out with. It is not an exhaustive showcase of DVC's features, so
+testers are encouraged to try other DVC repositories -- especially real-world
+cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Run `yarn setup:venv` from the
+  root of this repo to set up a virtual Python environment for the demo project
+  (in `demo/.env`).
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Important: Open the `demo/` folder of this repo directly in VS Code.
```

Comment:
hey @iterative/vs-code - do you primarily open it directly when you run the dev version or from the extension root?

Reply:
The normal process is to 

1. Open the project's root
2. Start the dev server
3. Hit f5 to start a debug session
4. Open the demo project in the debug session

Reply:
Thanks. From this I get that you tend to open the demo/ project directly when using it to test changes to the extension (which is `demo`'s main purpose).

> BTW, I'm having [this issue again](https://github.com/iterative/dvc-checkpoints-mnist/issues/20) in the demo/ project when the branch is not `main` -- I assume contributors will be in new branches when doing all this. I'll try to fix it and if not reach out in Slack for now.